src/api/report/submit-report.ts

```diff
@@ -53,6 +55,12 @@ export const uploadSavedReports = async () => {
     [] as Report[] | Promise<Report[]>,
   )
   localStorage.setItem(SAVED_REPORTS, JSON.stringify(unsuccessfulReports))
+
+  const uploadedReports = savedReports.length - unsuccessfulReports.length
+  createAlert({
+    type: AlertType.Success,
+    message: `${uploadedReports} reports were uploaded!`,
+  })
```

Comment:
@varCepheid the `uploadSavedReports` function runs 2 seconds after the initial load, and then every 30 seconds. Since there isn't a check here of uploadedReports !== 0, this ends up happening (a new one every 30 seconds):

![image](https://user-images.githubusercontent.com/13206945/215210563-e5867555-5178-4346-9a3a-4437114d6283.png)

The [deploy preview](https://deploy-preview-1523--peregrine.netlify.app/) that is automatically created for each PR is super helpful in testing out your changes! Peregrine has a lot of moving pieces and it is important to carefully test all code changes to make sure everything works as expected.